detectors/node/opentelemetry-resource-detector-aws/src/detectors/index.ts

```diff
@@ -14,7 +14,8 @@
  * limitations under the License.
  */
 
-export * from './AwsEc2Detector';
+export { awsEc2Detector } from './AwsEc2Detector';
```

Comment:
note for reviewer: small step forward for https://github.com/open-telemetry/opentelemetry-js/issues/4186